code/datums/gamemodes/gangwar.dm

```diff
@@ -1,3 +1,5 @@
+var/global/datum/hud/gang_victory/victory_hud
```

Comment:
why not use the singleton system instead? less pollution

Reply:
used a singleton, had to modify the victory hud so it didn't expect the winner in new but instead required a post-creation proc call